docs/api/IoT.js-API-HTTPS.md

```diff
@@ -0,0 +1,133 @@
+### Platform Support
+
+ The following shows Https module APIs available for each platform.
+
+|  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) | Tizen<br/>(Artik 10) |
+| :---: | :---: | :---: | :---: | :---: |
+| https.request  | X | X | X | O |
+| https.get  | X | X | X | O |
+
+
+# Https
+
+IoT.js provides HTTPS to support HTTPS clients enabling users to send HTTPS request easily.
+
+### https.request(options[, callback])
+* `options` {Object}
+  * `host` {string} A domain name or IP address of the server to issue the request to. Defaults to 'localhost'.
+  * `hostname` {string} Alias for host.
+  * `port` {number} Port of remote server. Defaults to 80.
+  * `method` {string} A string specifying the HTTPS request method. Defaults to 'GET'.
+  * `path` {string} Request path. Defaults to '/'. Should include query string if any. E.G. '/index.html?page=12'. An exception is thrown when the request path contains illegal characters. Currently, only spaces are rejected but that may change in the future.
+  * `headers` {Object} An object containing request headers.
+  * `auth` {string} Optional Basic Authentication in the form `username:password`. Used to compute HTTPS Basic Authentication header.
+  * `ca` {string} Optional file path to CA certificate. Allows to override system trusted CA certificates.
+  * `cert` {string} Optional file path to client authentication certificate in PEM format.
+  * `key1` {string} Optional file path to private keys for client cert in PEM format.
+* `callback` {Function}
+  * `response` {https.IncomingMessage}
+* Returns: {https.ClientRequest}
+
+Example:
+```javascript
+var https = require('https');
+
+var request = https.request({
+  method: 'POST',
+  port: 443,
+  headers: {'Content-Length': 3}
+});
+
+...
+
+request.end();
+```
+
+Note that in the example `req.end()` was called. With `https.request()` one must always call `req.end()` to signify that you're done with the request - even if there is no data being written to the request body.
+
+### https.get(options[, callback])
+* `options` {Object}
+* `callback` {Function}
+  * `response` {https.IncomingMessage}
+* Returns: {https.ClientRequest}
+
+Same as `https.request` except that `https.get` automatically call `req.end()` at the end.
```

Comment:
Could be a link. Options are also the same? Perhaps we should list them here for completeness.